pkg/component/plutono/plutono.go

```diff
@@ -84,6 +161,546 @@ func (p *plutono) WaitCleanup(ctx context.Context) error {
+						v1beta1constants.LabelNetworkPolicyToDNS:          v1beta1constants.LabelNetworkPolicyAllowed,
+						gardenerutils.NetworkPolicyLabel("logging", 3100): v1beta1constants.LabelNetworkPolicyAllowed,
+					}),
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name:            name,
+							Image:           p.values.Image,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							Env: []corev1.EnvVar{
+								{Name: "PL_ALERTING_ENABLED", Value: "false"},
+								{Name: "PL_SNAPSHOTS_EXTERNAL_ENABLED", Value: "false"},
+								{Name: "PL_AUTH_ANONYMOUS_ENABLED", Value: "true"},
+								{Name: "PL_USERS_VIEWERS_CAN_EDIT", Value: "true"},
+								{Name: "PL_DATE_FORMATS_DEFAULT_TIMEZONE", Value: "UTC"},
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "plutono-dashboards",
+									MountPath: "/var/lib/plutono/dashboards",
+								},
+								{
+									Name:      "plutono-datasources",
+									MountPath: "/etc/plutono/provisioning/datasources",
+								},
+								{
+									Name:      "plutono-dashboard-providers",
+									MountPath: "/etc/plutono/provisioning/dashboards",
+								},
+							},
+							Ports: []corev1.ContainerPort{
+								{
+									Name:          "web",
+									ContainerPort: int32(port),
+								},
+							},
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse("10m"),
+									corev1.ResourceMemory: resource.MustParse("32Mi"),
+								},
+								Limits: corev1.ResourceList{
+									corev1.ResourceMemory: resource.MustParse("400Mi"),
+								},
+							},
+						},
+					},
+					Volumes: []corev1.Volume{
+						{
+							Name: "plutono-dashboards",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: dashBoardConfigMapName,
+									},
+								},
+							},
+						},
+						{
+							Name: "plutono-datasources",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: dataSourceConfigMapName,
+									},
+								},
+							},
+						},
+						{
+							Name: "plutono-dashboard-providers",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: providerConfigMapName,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	if p.values.ClusterType == component.ClusterTypeSeed {
+		deployment.Labels = utils.MergeStringMaps(deployment.Labels, map[string]string{v1beta1constants.LabelRole: v1beta1constants.LabelMonitoring})
+		deployment.Spec.Template.Spec.AutomountServiceAccountToken = pointer.Bool(false)
```

Comment:
I think this can be set always - probably it was forgotten in the `seed-monitoring` charts